test/pde_systems/system_transformation.jl

```diff
@@ -0,0 +1,2 @@
+using MethodOfLines, ModelingToolkit, OrdinaryDiffEq, DomainSets
+using Test
```

Comment:
This is dead code?

Reply:
ah good catch, I had been running it direct I'll add to runtests